packages/aws-cdk-lib/aws-s3/lib/bucket.ts

```diff
@@ -1291,7 +1291,7 @@ export enum ObjectOwnership {
    */
   BUCKET_OWNER_PREFERRED = 'BucketOwnerPreferred',
   /**
-   * The uploading account will own the object.
+   * The bucket owner will own the object if the object is uploaded with the bucket-owner-full-control canned ACL. Without this setting and canned ACL, the object is uploaded and remains owned by the uploading account.
```

Comment:
The issue mentions `BUCKET_OWNER_PREFERRED`, but we are updating `OBJECT_WRITER`'s docstring here. Did you mean to update `BUCKET_OWNER_PREFERRED` instead?

Reply:
Oops.. Thanks for catching it ,I'll update it.